MeshTweaker.py

```diff
@@ -35,35 +70,51 @@ class Tweak:
      vector with R.
     And the relative unprintability of the tweaked object. If this value is
      greater than 10, a support structure is suggested.
-        """
+    """
+
+    def __init__(self, content, extended_mode=False, verbose=True, show_progress=False,
+                 favside=None, min_volume=False, parameter=None,  progress_callback=None):
+        # Load parameters
+        if parameter is None:
+            if min_volume:
+                parameter = PARAMETER_VOL
+            else:
+                parameter = PARAMETER
+
+        for k, v in parameter.items():
+            # print(f"{k} = {v}")
+            exec(f"self.{k} = {v}")
```

Comment:
This is really not the right way to do this (exec is dangerous, and the f"" notation isn't supported in python3.5 (which is what Cura runs on).

There are a few ways to do this a bit better though. The quickest one would be:
```python3
for k, v in parameter.items():
    setattr(self, k, v)
```

Reply:
Thanks!
I've made the required changes.
`setattr` is useful for this parameter extraction, because it is limited to the parameters only.

Reply:
I hope it can be merged now!